addons/account/models/account_move.py

```diff
@@ -3373,6 +3394,13 @@ def _compute_tax_audit(self):
 
             record.tax_audit = audit_str
 
+    def _get_not_entry_condition(self, aml):
+        """
+        Returns the condition to exclude entry move types
```

Comment:
Please be a little clearer on the fact that this is used to compute tax_audit :p